www/index.html

```diff
@@ -10,6 +10,10 @@
 </head>
 
 <body>
+  <select id="example">
+    <option value="BEW-HEL">BEW-HEL (filter usage)</option>
+    <option value="BEW-HEL-SEX">BEW-HEL-SEX (imperfect example)</option>
```

Comment:
Is imperfect example -> missing values example ?

Reply:
Well, at this point it's probably plain broken because the data is different in `integ` endpoint 😬

Reply:
I was wondering about the name :) it's not clear to me